backend.native/build.gradle

```diff
@@ -39,6 +41,7 @@ allprojects {
         maven { url kotlinCompilerRepo }
     }
 }
+task compilerVersionGeneration(type: VersionGenerator){}
```

Comment:
I think it's better to name it in the same manner as the protobuf generation task named. Something like `generateCompilerVersion`